keyboards/melgeek/z70ultra/z70ultra.c

```diff
@@ -18,6 +18,7 @@
 
 
 #ifdef RGB_MATRIX_ENABLE
+#ifdef REV1
```

Comment:
Everything from Lines 21-120 should be placed in a `keyboards/melgeek/z70ultra/rev1/rev1.c` file.